### PR TITLE
LGA-3093: Set up Google Analytics

### PR DESCRIPTION
**Pull request by BenMillar-MOJ**
### Enables Google Analytics
Attaches the Google Tag Manager code snippet to the head of the base template when analytics cookies are enabled.

### Attaches `gtm_anon_id` to GTM's dataLayer
To enable the `gtm_anon_id` to be read by Tag Manager the ID is attached, this ID is a UUID4 string which is stored in the session data and a cookie which lasts for 30 days.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by TawneeOwl — APPROVED** [review]
LGTM 🚢 I know Jav mentioned not needing to obfuscate the GTM id but this looks solid

File: app/templates/components/gtm.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{#
+This component enables Google Tag Manager.
+It should be included in the base template if the user has enabled tracking cookies.
+#}
+
+{% if "cookies_policy" in request.cookies %}
+{% if request.cookies.cookies_policy | dict | get_item("analytics") == "yes" %}
+
+{% set gtm_measurement_id = 'G-S8M7DQZ6N0' %}
```
Comment (digitalali-moj) [inline app/templates/components/gtm.html:9]: On GTM we have GTM-PJPFZVSS as the Tagman id for Access

Comment (BenMillar-MOJ) [reply app/templates/components/gtm.html]: Thanks Jav, great spot. This is resolved with the latest commit.

**Review by digitalali-moj — APPROVED** [review]
Rock star!

**Pull request merged**